oscore-cf/pom.xml

```diff
@@ -0,0 +1,31 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.californium</groupId>
+    <artifactId>parent</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>oscore-cf</artifactId>
+  <dependencies>
+	<dependency>
+		<groupId>com.augustcellars.cose</groupId>
+		<artifactId>cose-java</artifactId>
```

Comment:
the docs on GitHub indicate that
> Starting with version 0.9.7, the code was modified so that it only uses the JAVA cryptogprahic provider infrastructure rather than directly relying on the BouncyCastle implementations of these algorithms.

and further
> Some modes and algorithms are not generally supported by the default Cryptographic Provider. One example of this is the CCM mode for AES. Getting these algorithms to work requires that a new provider be installed as part of the application so that it is available for this module. 

Will your code work with Sun's/Oracle's standard Crypto Provider of Java 7?